src/database/populate.py

```
@@ -10,13 +11,13 @@
-    CountryModel,
+    CertificationModel,
     GenreModel,
-    ActorModel,
-    MoviesGenresModel,
-    ActorsMoviesModel,
-    LanguageModel,
-    MoviesLanguagesModel,
+    StarModel,
+    movie_genres,
+    movie_stars,
+    DirectorModel,
```

Ruff (0.11.9)

`database.DirectorModel` imported but unused

Remove unused import

(F401)